allenact/main.py

```diff
@@ -218,6 +212,25 @@ def get_argument_parser():
         "--version", action="version", version=f"allenact {__version__}"
     )
 
+    parser.add_argument(
+        "--distributed_ip_port",
+        dest="distributed_ip_port",
+        required=False,
+        type=str,
+        default="127.0.0.1:-1",
+        help="IP address and port of listener for distributed process with rank 0. "
+        "Port address -1 lets runner choose a free port.",
+    )
```

Comment:
Ah, now I see why the `"127.0.0.1:-1"` notation is nicer as it's easier to enter on the command line. Perhaps `distributed_ip_port` could be `distributed_ip_and_port` though?

Reply:
Can we add a TODO to point people (in the help) to the tutorial on this? Since it requires a bit of set up it would probably be useful

Reply:
👍